src/GRA.Web/Areas/PerformerRegistration/Views/Home/Information.cshtml

```diff
@@ -1,5 +1,7 @@
 ﻿@model GRA.Controllers.ViewModel.PerformerRegistration.Home.InformationViewModel
 
+@inject Microsoft.Extensions.Localization.IStringLocalizer<GRA.Resources.Shared> SharedLocalizer
```

Comment:
Localization probably isn't needed in PerformerRegistration.

Reply:
Fixed in commit 97b7d4757c5daf38b97e01a4b0f363bb290ee169